org.gnome.Crosswords.json

```diff
@@ -0,0 +1,70 @@
+{
+    "app-id" : "org.gnome.Crosswords",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "crosswords",
+    "finish-args" : [
+        "--device=dri",
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--env=PUZZLE_SET_PATH=/app/extensions/puzzle-sets"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/girepository-1.0",
+        "/lib/pkgconfig",
+        "/libexec/installed-tests",
+        "/man",
+        "/share/applications/org.gnome.Adwaita1.Demo.desktop",
+        "/share/metainfo/org.gnome.Adwaita1.Demo.metainfo.xml",
+        "/share/icons/hicolor/symbolic/apps/org.gnome.Adwaita1.Demo-symbolic.svg",
+        "/share/icons/hicolor/scalable/apps/org.gnome.Adwaita1.Demo.svg",
```

Comment:
```suggestion
```
Are those actually needed? You don't ship adwaita yourself as it is part of the sdk